src/App.css

```diff
@@ -1,5 +1,191 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
+
+body {
+  font-size: 1.3rem;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fafafa;
+  color: #121212;
+}
+
+/* #root {
+  border: 1px solid;
+  width: 60%;
+  margin: 10% auto;
+  background-color: #ffffff;
+} */
+
+li {
+  list-style: none;
+}
+
+.navbar nav ul {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  padding: 1% 5%;
+  background-color: #fff;
+}
```

Comment:
- [ ] Good job so far with your project you have a great design :+1:  But your need to work a bit on your navbar and make it looks like the Bookstore design in [Zeplin](https://zeplin.io/). :cry:  Please make sure that your navbar is Styling EXACTLY like Zeplin Bookstore's design, by adding **user icon** and `border-bottom` to your navbar.  it's very important to peer attention to details cause your future employer will judge your work on it. And following the zeplin design will make your application works fine and wonderful :smile: 
Please check the screenshots below to see the differences :arrow_down: 

#### Your design
![Screenshot from 2022-08-27 20-26-51](https://user-images.githubusercontent.com/6615394/187047016-f7be421d-034f-478f-8f7b-a777c63be237.png)

#### Zeplin design
![Screenshot from 2022-08-27 20-23-50(1)](https://user-images.githubusercontent.com/6615394/187047017-04868fd6-2baa-4763-807f-99d61ff6e2b8.jpg)